Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.yml

```diff
@@ -0,0 +1,21 @@
+commonfields:
+  id: ExtractAttackPattern
+  version: -1
+name: ExtractAttackPattern
+comment: Extract Attack Pattern Threat Intel Object.
```

Comment:
please add more description about this script. a specially where it is used in the chain and why.

Reply:
done